pkg/trace/api/api.go

```diff
@@ -87,6 +89,10 @@ func NewHTTPReceiver(conf *config.AgentConfig, dynConf *sampler.DynamicConfig, o
 	if features.Has("429") {
 		rateLimiterResponse = http.StatusTooManyRequests
 	}
+	appsecHandler, err := appsec.NewIntakeReverseProxy(conf.NewHTTPTransport())
+	if err != nil {
+		log.Error("appsec agent: ", err)
```

Comment:
All log messages need to be capitalised.

Reply:
Fixed

Reply:
Not 🙂 

Also, there is no "appsec agent". Rather say:
```suggestion
		log.Errorf("Could not instantiate AppSec: %v", err)
```